contracts/cw721-non-transferable/src/lib.rs

```diff
@@ -0,0 +1,85 @@
+pub use crate::msg::InstantiateMsg;
+use crate::state::{Config, CONFIG};
+use cosmwasm_std::Empty;
+pub use cw721_base::{
+    entry::execute as _execute, ContractError, Cw721Contract, ExecuteMsg, Extension,
+    InstantiateMsg as Cw721BaseInstantiateMsg, MintMsg, MinterResponse,
+};
+
+pub mod msg;
+pub mod state;
+
+// version info for migration info
+const CONTRACT_NAME: &str = "crates.io:cw721-non-transferable";
+const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
+
+pub type QueryMsg = cw721_base::QueryMsg<Empty>;
+pub type Cw721NonTransferableContract<'a> = Cw721Contract<'a, Extension, Empty, Empty, Empty>;
+
+#[cfg(not(feature = "library"))]
+pub mod entry {
+    use super::*;
+    use cosmwasm_std::entry_point;
+    use cosmwasm_std::{Binary, Deps, DepsMut, Env, MessageInfo, Response, StdResult};
```

Comment:
These two lines can be combined into a single statement

Reply:
fixed